packages/ui/package.json

```diff
@@ -46,7 +46,6 @@
     "dev": "",
     "format": "prettier . --write",
     "format:check": "prettier .",
-    "postinstall": "bun run build",
```

Comment:
bring this back

Reply:
Sorry for missing this out @SutuSebastian ,,, done!.